src/app.js

```diff
@@ -1,21 +1,24 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+
+  const {
+    pathname, searchParams,
+  } = new URL(req.url, `http://${req.headers.host}`);
+
+  const parts = pathname.split('/').filter(i => i);
```

Comment:
Do you really need to filter it like that? I think it's better to pass the more meaningful callback to make your code cleaner